ui/component/collectionItemsList/view.jsx

```diff
@@ -36,13 +36,10 @@ const CollectionItemsList = (props: Props) => {
     ...claimListProps
   } = props;
```

Comment:
In the props definittion, `doFetchItemsInCollection: ({}, ?() => void) => void,` needs an update since the callback was removed.